docs/reference/errors-and-warnings/NU1604.md

```diff
@@ -32,3 +32,22 @@ or
 > `<PackageReference Version="9.0.0" />`
 
 which implies a lower bound.
+
+If using [Central Package Managment](https://learn.microsoft.com/nuget/consume-packages/Central-Package-Management) you will need to update the `PackageVersion` `Version` attribute in your `Directory.Packages.Props` file.
```

Comment:
Normally this would be a different scenario. 

We'd have another Issue, Solution etc. 

We should also call out that this is a special case, since in a properly onboarded CPM scenario, you'd get NU1008.

Reply:
I wasn't sure if I should put it in another Solution section since it's the same error but yeah it's a special case so I can start with that

Reply:
Here's prior from both restore and pack.

* https://github.com/NuGet/docs.microsoft.com-nuget/blob/main/docs/reference/errors-and-warnings/NU1605.md
* https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128